src/components/Proposal/ProposalSummary/ProposalSummary.scss

```diff
@@ -198,6 +198,10 @@
   margin-right: 7px;
 }
 
+.bold {
+  font-weight: bold;
```

Comment:
how this is related to this pr ?

Reply:
this was to make the proposal ui good looking